src/test/java/org/gameshop/management/GameDaoTest.java

```diff
@@ -0,0 +1,55 @@
+package org.gameshop.management;
+
+import org.gameshop.management.mock.GameDaoMock;
+import org.gameshop.model.Game;
+import org.junit.Test;
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
The same suggestion to create and clear the DB, because every time you will get not an actual result